frontend/src/components/LeadRow.js

```diff
@@ -1,22 +1,34 @@
-import React from 'react';
+import React, { useState } from 'react';
 import { TableRow, TableCell, Box } from '@material-ui/core';
 import EditOutlinedIcon from '@material-ui/icons/EditOutlined';
 import DeleteOutlineOutlinedIcon from '@material-ui/icons/DeleteOutlineOutlined';
 import { useStyles } from './Home';
 
+const SocialMediaLink = ({ lead }) => {
+  const link =
+    lead['facebook'] ||
+    lead['linkedin'] ||
+    lead['twitter'] ||
+    lead['instagram'] ||
+    null;
+
+  return <a href={link}>{link}</a>;
```

Comment:
Just a heads up, this has the changes from #138 . Can you either make the [linting changes](https://github.com/codefordenver/partner-finder/pull/138#issuecomment-933987357) requested on that PR so it can be merged, or remove the changes that are not pertinent to this ticket?